### ISSUE TITLE
Add a `.reload()` to models

### ISSUE THREAD

**Issue by joakimnordling** [ioxiocom/firedantic#65]
It would be handy to have a `.reload()` on the models to re-fetch them from the DB instead of having to manually get the id and then load by id. This would be especially useful when writing unittests if you want to check an API modified a model as expected.

For reference [Arangodantic](https://github.com/ioxiocom/arangodantic/) has such functionality.

**Comment by antont** [comment]
Yep my tests do a lot of such refreshing too, so +1 for the need from here. No idea about the best design but maybe it's just such a reload method indeed.
